mllib/src/test/scala/org/apache/spark/ml/regression/GeneralizedLinearRegressionSuite.scala

```diff
@@ -495,8 +495,8 @@ class GeneralizedLinearRegressionSuite extends MLTest with DefaultReadWriteTest
        [1] 1.8121235  -0.1747493  -0.5815417
```

Comment:
Can you update the R script which generate the `deviance`?

Reply:
Updated. The updated script is sufficient to calculate deviance on its own.